npm/src/language-service/language-service.ts

```diff
@@ -151,6 +163,10 @@ export class QSharpLanguageService implements ILanguageService {
       if (url.protocol === qsharpLibraryUriScheme + ":") {
         code = wasm.get_library_source_content(url.pathname);
       }
+      if (!code) {
+        log.error(`expected ${url} to be in the library`);
+        return undefined;
+      }
```

Comment:
Seems like this should be inside of the previous `if` as it is specific to that branch.